### PR TITLE
More wizard workflows

### PR DESCRIPTION
**Pull request by robinschmid**
More workflows

- flow inject  DDA and library gen
- LC library gen
- add local csv database search as annotation method

### CODE REVIEW

File: src/main/java/io/github/mzmine/modules/tools/batchwizard/io/WizardWorkflowIOUtils.java [hunk 1]
```diff
@@ -208,11 +202,9 @@ public static void saveToFile(final List<WizardStepParameters> workflow, final F
   /**
    * Local files are save to {@link #getWizardSettingsPath()}
    *
-   * @param ALL_PRESETS all presets so that the correct object can be used
    * @return A list of local wizard workflows
    */
-  public static @NotNull List<LocalWizardWorkflowFile> findAllLocalPresetFiles(
-      final Map<WizardPart, List<WizardStepParameters>> ALL_PRESETS) {
+  public static @NotNull List<LocalWizardWorkflowFile> findAllLocalPresetFiles() {
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/tools/batchwizard/io/WizardWorkflowIOUtils.java:207]: Rename this file to WizardSequenceIOUtils and LocalWizardWorkflowFile to LocalWizardSequenceFile

File: src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/factories/IonMobilityWizardParameterFactory.java [hunk 2]
```diff
@@ -71,4 +71,18 @@ public WizardStepParameters create() {
...
+
+
+  /**
+   * Not all combinations work.
+   *
+   * @return supported combinations
+   */
+  public MassSpectrometerWizardParameterFactory[] getMatchingMassSpectrometerPresets() {
+    return switch (this) {
+      case TIMS, TWIMS ->
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/factories/IonMobilityWizardParameterFactory.java:83]: DTIMS here

**Review by SteffenHeu — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/mzmine/modules/tools/batchwizard/io/WizardWorkflowIOUtils.java (hunk 1 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/tools/batchwizard/io/WizardWorkflowIOUtils.java:207]: all renamed

File: src/main/java/io/github/mzmine/modules/tools/batchwizard/builders/BaseWizardBatchBuilder.java [hunk 3]
```diff
@@ -0,0 +1,1033 @@
...
+    param.setParameter(LocalCSVDatabaseSearchParameters.peakLists,
+        new FeatureListsSelection(FeatureListsSelectionType.BATCH_LAST_FEATURELISTS));
+    param.setParameter(LocalCSVDatabaseSearchParameters.dataBaseFile, csvLibraryFile);
+    param.setParameter(LocalCSVDatabaseSearchParameters.fieldSeparator,
+        csvLibraryFile.getName().toLowerCase().endsWith(".csv") ? "," : "\t");
+    param.setParameter(LocalCSVDatabaseSearchParameters.columns, csvColumns);
+    param.setParameter(LocalCSVDatabaseSearchParameters.mzTolerance, mzTolInterSample);
+    param.setParameter(LocalCSVDatabaseSearchParameters.rtTolerance,
+        Objects.requireNonNullElse(rtTol, new RTTolerance(9999999, Unit.MINUTES)));
+    param.setParameter(LocalCSVDatabaseSearchParameters.mobTolerance, new MobilityTolerance(0.01f));
```
[the remark this answers is not in the document]
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/tools/batchwizard/builders/BaseWizardBatchBuilder.java:1015]: replaced with IMS FWHM

**Review by robinschmid — COMMENTED** [review]
All done

**Review by SteffenHeu — APPROVED** [review]

**Pull request merged**